runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -149,5 +155,18 @@ else if (currentHP > maxHP)
 			// Show it going down
 			hitpointsPercentage = 1 - hitpointsPercentage;
 		}
+
+		if (config.getNotifyBeforeHpRegenSeconds() > 0 && currentHP < maxHP && shouldNotifyHpRegenThisTick(ticksPerHPRegen))
+		{
+			notifier.notify("[" + client.getLocalPlayer().getName() + "] regenerates their next hitpoint soon!");
+		}
+	}
+
+	private boolean shouldNotifyHpRegenThisTick(int ticksPerHPRegen)
+	{
+		// if the configured duration lies between two ticks, choose the earlier tick
+		final int ticksBeforeHPRegen = ticksPerHPRegen - ticksSinceHPRegen;
+		final int notifyTick = (int) Math.ceil(config.getNotifyBeforeHpRegenSeconds() / HP_REGEN_TICK_LENGTH);
```

Comment:
This is basically game tick, don't we have constant for this somewhere? If no, then this is good opportunity to add it to Constants file

Reply:
Yeah, I noticed this. I wasn't sure whether to add it seeing that a few plugins already define their own version of the 0.6s tick constant in a few places.

If I do this, should I also update the other plugins to use the const from Constants within this PR? I could add that change as a seperate commit to this one and edit the name of the PR to reflect this.

Reply:
A separate commit would be good, but that sounds like a good idea.